jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -286,9 +286,9 @@ public void setupAdvancedSearchView() {
 
         view.addTextFilter(constants.Id(),
                            constants.FilterByErrorId(),
-                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_ERROR_ID,
+                           v -> addAdvancedSearchFilter(likeTo(COLUMN_ERROR_ID,
```

Comment:
Would this also work if "_" operator is used?

Reply:
@bsremac it should respect the wildcards usage as any other field using _likeTo_,  I believe both "_" and "%" should work just fine.